src/transformers/tokenization_bart.py

```diff
@@ -110,6 +110,12 @@ class MBartTokenizer(XLMRobertaTokenizer):
     id_to_lang_code = {v: k for k, v in lang_code_to_id.items()}
     cur_lang_code = lang_code_to_id["en_XX"]
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fairseq_tokens_to_ids.update(self.lang_code_to_id)
+        self.fairseq_ids_to_tokens = {v: k for k, v in self.fairseq_tokens_to_ids.items()}
+        self._additional_special_tokens = list(self.lang_code_to_id.keys())
```

Comment:
allows language codes to be filtered out by `tokenizer.decode(skip_special_tokens=True)`